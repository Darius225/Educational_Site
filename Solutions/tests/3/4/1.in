100
28637 13116 5500 29009 32515 14223 22903 32713 23711 15085 21768 27990 8061 19199 10501 14850 24529 5694 9429 10024 2775 262 22323 5419 6583 4332 7216 26983 12203 8786 19575 4200 25443 7225 29141 28762 14249 5718 9465 24513 1857 9575 30840 29803 26643 16839 5396 16034 18828 31808 27453 17574 21057 29985 7640 21082 16597 18833 2624 28989 21533 27427 4490 7324 32387 5547 1806 1542 26792 22351 26658 32257 1870 24289 16681 21755 18548 1132 25549 18563 7106 16396 21128 20665 23080 771 1385 25091 20675 9994 27669 30466 29440 26597 20748 30405 30723 6970 557 19260 