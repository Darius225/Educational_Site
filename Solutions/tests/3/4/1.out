262 557 771 1132 1385 1542 1806 1857 1870 2624 2775 4200 4332 4490 5396 5419 5500 5547 5694 5718 6583 6970 7106 7216 7225 7324 7640 8061 8786 9429 9465 9575 9994 10024 10501 12203 13116 14223 14249 14850 15085 16034 16396 16597 16681 16839 17574 18548 18563 18828 18833 19199 19260 19575 20665 20675 20748 21057 21082 21128 21533 21755 21768 22323 22351 22903 23080 23711 24289 24513 24529 25091 25443 25549 26597 26643 26658 26792 26983 27427 27453 27669 27990 28637 28762 28989 29009 29141 29440 29803 29985 30405 30466 30723 30840 31808 32257 32387 32515 32713 